static/js/unread_ops.js

```diff
@@ -227,3 +227,10 @@ export function mark_topic_as_read(stream_id, topic, cont) {
         success: cont,
     });
 }
+
+export function mark_pm_as_read(user_ids_string) {
+    // unread_ids_string is a list of user ids which are participants
+    // in the conversation other than the current user. Eg: "123,124" or "123"
+    const unread_msg_ids = unread.get_msg_ids_for_person(user_ids_string);
```

Comment:
This looks correct but perhaps we should add a prep commit to rename `get_msg_ids_for_person` to `unread.get_msg_ids_for_user_ids_string`? The current name suggests it doesn't handle group PMs.